guides/doc-Upgrading_and_Updating/topics/upgrading_external_database.adoc

```diff
@@ -0,0 +1,21 @@
+[id="Upgrading_External_Database_{context}"]
+= Upgrading External Database
+
+You can upgrade an external database from {RHEL} 7 to {RHEL} 8 while upgrading {Project} from {ProjectVersionPrevious} to {ProjectVersion}.
+
+.Prerequisites
+Create a new {RHEL} 8 based host for PostgreSQL server that follows the external database on {RHEL} 8 documentation.
+
+.Procedure
+. Copy the backup database to the new host.
+. Restore the backup database on the new host.
+
+If you keep the same hostname on the new PostgreSQL server, you need to update the DNS preventing the need to do  updates on {Project} itself.
```

Comment:
```suggestion
If you keep the same hostname on the new PostgreSQL server, you need to update the DNS preventing the need to do updates on {Project} itself.
```

I am unsure I understand this correctly. Is this a prerequisite?

Reply:
Wasn't sure if this needed to be a step but it's not a prerequisite. Since it is telling the user to update the DNS as a result of keeping the host name, I am changing it to "."